tests/unit/components/test_dataset_storage.py

```diff
@@ -108,6 +114,7 @@ def test_info_transform(self, fxt_stream_extractor: MagicMock, fxt_infos: Datase
         storage.transform(ProjectInfos, dst_infos=dst_infos)
 
         assert storage.infos().get("new") == "info"
+        assert fxt_stream_extractor.__iter__.call_count == 0
```

Comment:
I have one quick question. If it was `DatasetStorage`, `storage` was created from the `fxt_extractor` and it called the `infos()`, which may run `init_cache()`. If so, all items within `fxt_extractor` is also called? What I asked is, `fxt_extractor.__iter__.call_count == 1` for `DatasetStorage`?

Reply:
Yes